Doc/extending/extending.rst

```diff
@@ -127,10 +127,10 @@ Intermezzo: Errors and Exceptions
 
 An important convention throughout the Python interpreter is the following: when
 a function fails, it should set an exception condition and return an error value
-(usually a ``NULL`` pointer).  Exceptions are stored in a static global variable
-inside the interpreter; if this variable is ``NULL`` no exception has occurred.  A
-second global variable stores the "associated value" of the exception (the
-second argument to :keyword:`raise`).  A third variable contains the stack
+(usually a ``NULL`` pointer).  Exception information is stored in 3 static
+global variables inside the interpreter.  The first is the exception type,
+or ``NULL`` if no exception has occurred.  The second is the exception
+instance.  The third variable contains the stack
 traceback in case the error originated in Python code.  These three variables
```

Comment:
it could originate in C but then go to python and get a traceback, no?

Reply:
This is, actually, routine, as in IndexError originating in the C list code when executing, for instance, '[][1]'.  So the author (apparently Georg Brandl) must have meant something else.  I'm deleting the statement.